src/main/java/tvprogramme/TvProgrammeController.java

```diff
@@ -0,0 +1,24 @@
+package io.apiapp.springbootstarter.tvprogramme;
```

Comment:
package name should be the same as path, so this file should live in `src/main/java/io/apiapp/springbootstrater/tvprogramme`